tools/footprint/footprint.xml

```diff
@@ -39,16 +39,16 @@
             input_dir=\$(python -c "import os; print(os.path.dirname('$bam_file'))")
 
             &&
-            mv \$(find \$input_dir -name "*.PARAM") PARAM
+            mv \$input_dir/*.PARAM PARAM
```

Comment:
File "/tmp/tmp170mOf/galaxy-dev/lib/galaxy/tools/evaluation.py", line 454, in build
    raise e
NotFound: cannot find 'input_dir'